specification/communication/data-plane/PhoneNumbers/preview/2022-12-02-preview2/examples/GetPhoneNumber.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "api-version": "2022-12-02-preview2",
+    "endpoint": "https://resourcename.communication.azure.com/",
+    "phoneNumber": "+11234567890"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "11234567890",
+        "phoneNumber": "+11234567890",
+        "countryCode": "US",
+        "assignmentType": "user",
+        "phoneNumberType": "geographic",
+        "capabilities": {
+          "calling": "inbound+outbound",
+          "sms": "inbound"
+        },
+        "purchaseDate": "2020-11-19T16:31:49.048Z",
+        "cost": {
+          "amount": 0.8,
+          "currencyCode": "USD",
+          "billingFrequency": "monthly"
+        },
+        "operatorName": "Microsoft",
+        "operatorId": "d2fea165-4540-4ec8-a93f-6ed0baca28cd",
+        "phoneNumberSource": "cloud"
+      }
```

Comment:
TollFreeVerificationStatus property is missing.

Reply:
Added, thank you!